private-templates-service/client/src/components/AdaptiveCardPanel/index.ts

```diff
@@ -0,0 +1,3 @@
+import AdaptiveCardPanel from "./AdaptiveCardPanel";
+
+export default AdaptiveCardPanel;
```

Comment:
\n

Reply:
you can tell if there is a new line or not if there is a 
![image](https://user-images.githubusercontent.com/19780117/73896797-e1b00880-4838-11ea-92e4-20dec513606a.png)

Reply:
there is a newline there

Reply:
:o